packages/@dcl/sdk-commands/src/commands/start/server/endpoints.ts

```diff
@@ -483,48 +547,65 @@ async function fakeEntityV3FromProject(
   const projectFiles = await getProjectPublishableFilesWithHashes(components, project.workingDirectory, hashingFunction)
   const contentFiles = projectFilesToContentMappings(project.workingDirectory, projectFiles)
 
-  if (project.kind === 'scene') {
-    const sceneJsonPath = path.resolve(project.workingDirectory, 'scene.json')
-    const sceneJson = JSON.parse(await components.fs.readFile(sceneJsonPath, 'utf-8'))
-    const { base, parcels }: { base: string; parcels: string[] } = sceneJson.scene
-    const pointers = new Set<string>()
-    pointers.add(base)
-    parcels.forEach(($) => pointers.add($))
+  if (project.kind === 'scene' || project.kind === 'smart-wearable') {
+    return createSceneEntity(components, project, contentFiles, hashingFunction)
```

Comment:
for `project.kind === 'smart-wearable'` shouldn't we call `createWearableEntity`?